.github/workflows/build-linux-prod.yml

```diff
@@ -0,0 +1,45 @@
+name: linux production build
+on:
+  push:
+    tags:
+      - "v*"
```

Comment:
do you think it is possible to also constraints on master? or it makes no sense? thanks